src/devana/syntax_abstraction/typeexpression.py

```diff
@@ -557,11 +565,18 @@ def modification(self) -> TypeModification:
                 type_source = TypeExpression.cursor_parse_from_pointer(type_c)
             else:
                 type_source = type_c.get_pointee()
+
+        if type(self._cursor) is cindex.Cursor:
```

Comment:
src/devana/syntax_abstraction/typeexpression.py:569:11: C0123: Use isinstance() rather than type() for a typecheck. (unidiomatic-typecheck)